### PR TITLE
chore: add code of conduct and pr template

### PR DESCRIPTION
**Pull request by luciorubeens**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## Proposed changes
+
+Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
+
+## Types of changes
+
+What types of changes does your code introduce to Appium?
```
Comment (ItsANameToo) [inline .github/PULL_REQUEST_TEMPLATE.md]: Appium 🤔

Comment (faustbrian) [reply .github/PULL_REQUEST_TEMPLATE.md]: Lucio, pay attention 👀

Comment (luciorubeens) [reply .github/PULL_REQUEST_TEMPLATE.md]: 🙈

**Comment by faustbrian** [comment]
I am really not a fan of those `Code of Conducts`. Do we really need one on top of our contributing guides and PR templates?

**Comment by luciorubeens** [comment]
not necessarily, it's just a [recommended community standards](https://github.com/ArkEcosystem/core/community).

**Pull request merged**